packages/testing/src/test/chai-retry-plugin.unit.ts

```diff
@@ -0,0 +1,290 @@
+import Chai, { expect } from 'chai';
+import { sleep } from 'promise-assist';
+import { chaiRetryPlugin } from '../chai-retry-plugin/chai-retry-plugin';
+
+Chai.use(chaiRetryPlugin);
+
+const getFunctionToRetry = (func: (attempts: number) => unknown) => {
```

Comment:
use more generic name. I suggest `withCallCount`.
I'd say that approach when function returned (not object) is a bit cleaner:
```
function withCallCount(f) {
   const wrapperFunction = () => {
       wrapperFunction.callCount++;
       return f(wrapperFunction.callCount);
   }
    wrapperFunction.callCount = 0;
    
    return wrapperFunction;
}
```

Reply:
let's put this helper at the bottom of the file

Reply:
@yurii-ve fixed